app/src/main/java/com/infomaniak/mail/data/cache/MailRealm.kt

```diff
@@ -113,25 +132,36 @@ object MailRealm {
 
         private const val APP_SETTINGS_DB_NAME = "AppSettings.realm"
         private const val MAILBOX_INFO_DB_NAME = "MailboxInfo.realm"
-        private fun MAILBOX_CONTENT_DB_NAME(currentMailboxId: Int) = "${AccountUtils.currentUserId}-${currentMailboxId}.realm"
-
-        val appSettings = RealmConfiguration
-            .Builder(RealmSets.appSettings)
-            .name(APP_SETTINGS_DB_NAME)
-            .deleteRealmIfMigrationNeeded()
-            .build()
-
-        val mailboxInfo = RealmConfiguration
-            .Builder(RealmSets.mailboxInfo)
-            .name(MAILBOX_INFO_DB_NAME)
-            .deleteRealmIfMigrationNeeded()
-            .build()
-
-        fun mailboxContent(currentMailboxId: Int) = RealmConfiguration
-            .Builder(RealmSets.mailboxContent)
-            .name(MAILBOX_CONTENT_DB_NAME(currentMailboxId))
-            .deleteRealmIfMigrationNeeded()
-            .build()
+        private fun MAILBOX_CONTENT_DB_NAME(mailboxId: Int) = "${AccountUtils.currentUserId}-${mailboxId}.realm"
+        private val CONTACTS_DB_NAME get() = "${AccountUtils.currentUserId}.realm"
```

Comment:
I think it'll better to also add the db name, for example `contacts-XXX.realm`, `mailbox-XXX-XXX.realm`